modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/DataLayoutResourceTest.java

```diff
@@ -349,6 +366,40 @@ public void testPostDataLayoutContext() throws Exception {
 		Assert.assertEquals(200, httpResponse.getStatusCode());
 	}
 
+	@Override
+	@Test
+	public void testPutDataLayout() throws Exception {
+		super.testPutDataLayout();
+
+		DataLayout postDataLayout = testPutDataLayout_addDataLayout();
```

Comment:
this should be `putDataLayout`

Reply:
Despite the method name is `testPutDataLayout_addDataLayout`, it calls the Data Layout post method (because it is adding a new Data Layout). What do you think?

Reply:
I think it's very weird Oo Let's go with the putDataLayout anyway

Reply:
The issue here is the prefix/suffix of the utility method being used. The first part, before the underscore in `testPutDataLayout_addDataLayout` is where this utility method is being used, while the second part is what it really does. I've seen this naming pattern in other test classes. So, if we are going to follow naming conventions, I would change the variable name to `addedDataLayout` or leave it prefixed by `post`.

Reply:
Or perhaps, just leave it as `dataLayout`?